backend/src/email/routes/email-campaign.routes.ts

```diff
@@ -88,6 +88,21 @@ const duplicateCampaignValidator = {
   }),
 }
 
+const tesseractCampaignValidator = {
+  [Segments.BODY]: Joi.object({
+    url: Joi.string()
+      .trim()
+      .custom((value: string, helpers: any) => {
+        const url = value.match(/^https:\/\/storage(-test)?.vault.gov.sg\/.+$/g)
```

Comment:
If we already have this, do we still need `isValidVaultUrl` later? There is a little overlap between what both of this does (checking for permitted hostname). If we need to keep this, perhaps we can add this to the config file to make it easier to test locally with another bucket/host.